docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,727 @@
+      - `kubectl config use-context $CONTEXT  --cluster=$CONTEXT`
+
+Next, make a kubeconfig file for the kubelets and kube-proxy.  There are a couple of options for how 
+many distinct files to make:
+  1. Use the same credential as the admin
+    - This is simplest to setup.
+  1. One token and kubeconfig file for all kubelets, one for all kube-proxy, one for admin.
+    - This mirrors what is done on GCE today
+  1. Different credentials for every kubelet, etc.
+    - We are working on this but all the pieces are not ready yet.
+
+You can make the files by copying the `$HOME/.kube/config`, by following the code
+in `cluster/gce/configure-vm.sh` or by using the following template:
+```
+apiVersion: v1
+kind: Config
+users:
+- name: kubelet
+  user:
+    token: ${KUBELET_TOKEN}
+clusters:
+- name: local
+  cluster:
+    certificate-authority-data: ${CA_CERT_BASE64_ENCODED}
+contexts:
+- context:
+    cluster: local
+    user: kubelet
+  name: service-account-context
+current-context: service-account-context
+```
+Put the kubeconfig(s) on every node.  The examples later in this
+guide assume that there are kubeconfigs in `/var/lib/kube-proxy/kubeconfig` and
+`/var/lib/kubeletproxy/kubeconfig`.
+
+## Configuring and Installing Base Software on Nodes
+
+This section discusses how to configure machines to be kuberetes nodes. 
+
+You should run three daemons on every node:
+  - docker or rkt
+  - kubelet
+  - kube-proxy
+
+You will also need to do assorted other configuration on top of a
+base OS install.
+
+Tip: One possible starting point is to setup a cluster using an existing Getting
+Started Guide.   After getting a cluster running, you can then copy the init.d scripts or systemd unit files from that
+cluster, and then modify them for use on your custom cluster.
+
+### Docker
+The minimum required Docker version will vary as the kubelet version changes.  The newest stable release is a good choice.  Kubelet will log a warning and refuse to start pods if the version is too old, so pick a version and try it.
+
+If you previously had Docker installed on a node without setting Kubernetes-specific
+options, you may have a Docker-created bridge and iptables rules.  You may want to remove these
+as follows before proceeding to configure Docker for Kubernetes.
+```
+iptables -t nat -F 
+ifconfig docker0 down
+brctl delbr docker0
+```
+
+The way you configure docker will depend in whether you have chosen the routable-vip or overlay-network approaches for your network.
+Some docker options will want to think about:
+  - `--bridge cbr0`
+  - `--iptables=false`
+  - `--ip-masq=false`
+    - if have setup PodIPs to be routable, then you want this false, otherwise, docker will
+      rewrite the PodIP source-address to a NodeIP, to allow the Pod to talk to IPs outside the
+      cluster.
+    - if you are using an overlay network, consult those instructions.  
+  - `--bip=` and `--mtu=`
+    - may be required when using Flannel; see Flannel instructions.
```

Comment:
mtu may be required when using flannel because of udp encapsulation but i believe --bip should always be the cidr of the PodNet for that specific node, otherwise the bridge will come up with the default docker cidr.